### PR TITLE
feat(all): add prefix t- to internal classes to avoid clashed

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [x] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

## Other information
addon-charts, addon-commerce, addon-doc, addon-editor, addon-mobile

### CODE REVIEW

File: projects/addon-charts/components/axes/axes.template.html [hunk 1]
```diff
@@ -1,93 +1,97 @@
-<div *ngIf="hasYLabels" class="side" [class.side_padding]="hasXLabels">
+<div *ngIf="hasYLabels" class="t-side" [class.side_padding]="hasXLabels">
```
Comment (waterplea) [inline projects/addon-charts/components/axes/axes.template.html:1]: `[class.t-side_padding]`, gotta watch out for these bindings :D

File: projects/addon-charts/components/axes/axes.template.html [hunk 2]
```diff
@@ -1,93 +1,97 @@
...
             {{fallback(label)}}
         </div>
     </div>
 </div>
-<div *ngIf="hasYSecondaryLabels" class="side" [class.side_padding]="hasXLabels">
-    <div *ngIf="!axisYSecondaryInset" class="labels-y labels-y_secondary">
+<div
+    *ngIf="hasYSecondaryLabels"
+    class="t-side"
+    [class.side_padding]="hasXLabels"
```
Comment (waterplea) [inline projects/addon-charts/components/axes/axes.template.html:80]: 👀

File: projects/addon-charts/components/bar-chart/bar-chart.template.html [hunk 3]
```diff
@@ -1,20 +1,20 @@
 <div
     *ngFor="let set of transposed; let index = index"
     tuiHintDirection="top-left"
-    class="wrapper"
+    class="t-wrapper"
     [class.wrapper_hoverable]="hasHint"
```
Comment (waterplea) [inline projects/addon-charts/components/bar-chart/bar-chart.template.html:5]: 👀

**Review by waterplea — CHANGES_REQUESTED** [review]
Watch out for bindings!

**Review by nsbarsukov — APPROVED** [review]

File: projects/addon-mobile/internal/primitive-calendar-mobile/primitive-calendar-mobile.style.less [hunk 4]
```diff
@@ -110,13 +110,13 @@
 }
 
 @media screen and (max-width: 22.4375rem) {
-    .cell,
-    :host._ios .cell {
+    .t-cell,
+    :host._ios .t-cell {
```
Comment (waterplea) [inline projects/addon-mobile/internal/primitive-calendar-mobile/primitive-calendar-mobile.style.less:114]: Please also check host binging classes. Might be in a form of:
```
    host: {
        '[class._ios]': 'isIOS',
    },
```

**Review by waterplea — APPROVED** [review]

File: projects/addon-mobile/internal/primitive-calendar-mobile/primitive-calendar-mobile.style.less (hunk 4 above)
Comment (vladimirpotekhin) [reply projects/addon-mobile/internal/primitive-calendar-mobile/primitive-calendar-mobile.style.less:114]: We discussed this and decided not to touch these classes.

**Pull request merged**